classes/boostnavbar.php

```diff
@@ -0,0 +1,187 @@
+<?php
```

Comment:
We should put the regular moodle licence at the top + comment for class with copyright, author, etc...